builder/package.ubuntu-1804

```diff
@@ -0,0 +1,50 @@
+if [[ ! -d /tmp/output/ubuntu-1804 ]]; then
+  mkdir /tmp/output/ubuntu-1804
+fi
+
+fpm \
+  -s dir \
+  -t deb \
+  -v ${BUILD_NO} \
+  -n R-${R_VERSION} \
+  -p /tmp/output/ubuntu-1804/ \
+  --prefix=/opt/R \
+  -d zip \
+  -d unzip \
+  -d libpaper-utils \
+  -d xdg-utils \
+  -d libblas3 \
+  -d libatlas3-base \
+  -d libblas3 \
```

Comment:
Some of these packages seem to be duplicated? But now that we have OpenBLAS, we could remove these other BLAS/LAPACK libraries - libatlas, libblas, and liblapack.

Reply:
✅ Done